recipes/circleci-helpers/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "circleci-helpers" %}
+{% set version = "0.1.1" %}
+{% set checksum = "21e3d210de493bebce598e0209aebc588ebe16cab4be17de2684590649aace6e" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ checksum }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record=record.txt
+  entry_points:
+    - circle-matrix = circleci_helpers.matrix.main:main
+
+requirements:
+  build:
+    - python
+    - pip
+    - setuptools
```

Comment:
Yeah, but I'd prefer to see direct dependency appear in this list. [`setuptools`]( https://github.com/dennybaa/circleci-helpers/blob/4a93ed20d7eae0950f73227a9ad85873ce79ea2e/setup.py#L4 ) is one.